test/fail_compilation/issue19969.d

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
I think this test is not necessary after all. It was created before I see all the changes required in fail_arrayop1.d